package.json

```diff
@@ -64,20 +66,22 @@
   },
   "scripts": {
     "analyze": "yarn build && source-map-explorer 'build/static/js/*.js' --gzip",
-    "build": "yarn install && env $(yarn setenv) vite build && rm build/config.json",
+    "build": "tsc --build && env $(yarn setenv) vite build && rm build/config.json",
     "lint": "eslint --fix --max-warnings=0 .",
     "optimize-image-svgs": "svgo --enable=inlineStyles,prefixIds --config '{ \"plugins\": [ { \"inlineStyles\": { \"onlyMatchedOnce\": false } }] }' --pretty -f src/images -r -p 1 --multipass",
     "preinstall": "node .hooks/check-engine-light.js",
     "setenv": "echo VITE_APP_VERSION=$(git rev-parse HEAD) VITE_APP_BUILD_TIMESTAMP=$(date -u \"+%s000\")",
-    "test": "TZ=UTC jest",
-    "check-types": "check-dts types/**/*.ts",
+    "test": "jest",
+    "check-dts": "check-dts types/**/*.ts",
+    "check-types": "yarn workspaces foreach run check-dts",
```

Comment:
`@terra-ui-packages/core-utils` includes type helpers and also has a `check-dts` script to check those.